lib/drawers/publish-page.vue

```diff
@@ -220,6 +52,13 @@
       <span class="archive-help">You may archive any page that isn't published (or scheduled to be published). Archived pages will not show up in the Clay Menu unless you explicitly filter for them.</span>
       <ui-button class="archive-submit" buttonType="button" type="primary" color="red" :disabled="isScheduled || isPublished || isArchived" @click.stop="archivePage(true)">Archive</ui-button>
     </ui-collapsible>
+
+
+    <!-- restore from published -->
+    <ui-collapsible class="publish-section publish-archive" title="Restore Published Version">
+      <span class="archive-help">You may override ALL local changes to the page by Restoring the Page to the Published Version.  This can not be undone.</span>
```

Comment:
How about something like:


> You may override all changes to this page by restoring to the most recently published version of the page by clicking the button below. **Warning:** this action cannot be undone, you will lose all changes applied to this post since the last publish.

I'm mainly worried the word `local` won't translate. Once a user makes a change in clay it's technically global since any user who comes in between edits will see what has already been done.

Reply:
I'll update the copy